app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js

```diff
@@ -211,6 +213,12 @@ export function testCleanup(container, app) {
   resetModelTransformers();
   resetMentions();
   cleanupTemporaryModuleRegistrations();
+  cleanupCssGeneratorTags();
+}
+
+function cleanupCssGeneratorTags() {
```

Comment:
This must be done because these tags are added in the `head` of the document rather than the ember root element.